test/hitchhiker/konserve_test.clj

```diff
@@ -0,0 +1,69 @@
+(ns hitchhiker.konserve-test
+  (:require [clojure.core.rrb-vector :refer [catvec]]
+            [clojure.test :refer :all]
+            [clojure.test.check.clojure-test :refer [defspec]]
+            [clojure.test.check.generators :as gen]
+            [clojure.test.check.properties :as prop]
+            [konserve.filestore :refer [new-fs-store delete-store]]
+            [hitchhiker.konserve :as kons]
+            [hitchhiker.tree.core :as core]
+            hitchhiker.tree.core-test
+            [hitchhiker.tree.messaging :as msg]
+            [clojure.core.async :refer [<!! promise-chan]]))
+
```

Comment:
I have done so and found a bug in one of the deserializers :). I really like spec, I should use it more...